chain/chain/src/chain.rs

```diff
@@ -4193,8 +4193,11 @@ impl<'a> ChainUpdate<'a> {
         provenance: &Provenance,
         on_challenge: &mut dyn FnMut(ChallengeBody),
     ) -> Result<(), Error> {
-        // Refuse blocks from the too distant future.
-        if header.timestamp() > Clock::utc() + Duration::seconds(ACCEPTABLE_TIME_DIFFERENCE) {
+        // Refuse blocks from the too distant future. Only exception to this is while we're within
+        // sandbox, we're allowed to jump to the future via fast forwarding.
+        if cfg!(feature = "sandbox")
```

Comment:
shouldn't this be `!cfg!(feature = "sandbox")`?

Reply:
Yup, thanks for pointing it out!